### PR TITLE
sigverif: add pt_BR translation

### PR DESCRIPTION
**Pull request by isrmicha**
adding pt_BR translation to windows/sigverif.md

- [ ] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.pt_BR/windows/sigverif.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# sigverif
+
+> Uma interface gráfica para verificação de assinaturas de arquivos do sistema.
+
+- Abre a itnerface de verificação de assinatura.
```
Comment (navarroaxel) [inline pages.pt_BR/windows/sigverif.md:5]:
```suggestion
- Abre a interface de verificação de assinatura.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @isrmicha, welcome and thanks for your submission. I left 1 comment.

Remember that you can translate more than 1 page in 1 PR, just send 1 commit per file. (no more than 10 translation per PR).
1 PR per page is only required for the primary (English) pages

**Review by schneiderl — APPROVED** [review]
Hi, 
Thank you for your PR. It is really appreciated. 
It looks good to me after @navarroaxel comment is addressed.

**Review by isrmicha — COMMENTED** [review]
Thanks for the review

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**